console/v1/types_console_cli_download.go

```diff
@@ -21,7 +21,15 @@ type ConsoleCLIDownloadSpec struct {
 	// description is the description of the CLI download (can include markdown).
 	Description string `json:"description"`
 	// links is a list of objects that provide CLI download link details.
-	Links []Link `json:"links"`
+	Links []CLIDownloadLink `json:"links"`
+}
+
+type CLIDownloadLink struct {
+	// text is the display text for the link
+	// +optional
+	Text string `json:"text"`
```

Comment:
https://github.com/openshift/console-operator/blob/master/manifests/00-crd-extension-console-cli-download.yaml#L55

At present, only `href` is required. The new generated CRD should continue to maintain this level of validation.

Reply:
However, this can actually lead to a poor user experience as the `defaulted` text is ambiguous if the user provides more than one link:

![Screen Shot 2019-10-02 at 4 43 44 PM](https://user-images.githubusercontent.com/280512/66080321-dd46ac80-e533-11e9-8425-624309ccbd55.png)

Just noting this, it is not a hard problem to fix if the user is paying attention. 
@spadgett